centreon/src/Core/AgentConfiguration/Infrastructure/Serializer/ConfigurationParametersInterfaceNormalizer.php

```diff
@@ -58,10 +58,23 @@ public function normalize(
     }
 
     /**
-     * @inheritDoc
+     * @param array<string, mixed> $context
+     * @param mixed $data
+     * @param ?string $format
```

Comment:
```suggestion
     * @param mixed $data
     * @param ?string $format
     * @param array<string, mixed> $context
```
just to align phpdoc params order with method signature